app/experimenter/experiments/forms.py

```diff
@@ -42,13 +42,12 @@ def clean(self, value):
 
 
 class BugzillaURLField(forms.URLField):
-    BUGZILLA_BASE_URL = "https://bugzilla.mozilla.org/"
 
     def clean(self, value):
         cleaned_value = super().clean(value)
 
         if cleaned_value:
-            if self.BUGZILLA_BASE_URL not in cleaned_value:
+            if settings.BUGZILLA_HOST not in cleaned_value:
                 raise forms.ValidationError(
                     "Please provide a valid Bugzilla URL"
```

Comment:
We have to include the host in the error message otherwise people on dev/stage/local won't understand why it's not valid.